applications/nrf_desktop/doc/ble_latency.rst

```diff
@@ -39,6 +39,9 @@ You can set the option :ref:`CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK <config_desktop
 This speeds up sending the first HID report after not sending a report for some connection intervals.
 Enabling this option increases the power consumption - the connection latency is kept low unless the device is in the low power mode.
 
+You can use the :ref:`CONFIG_DESKTOP_BLE_LATENCY_PM_EVENTS <config_desktop_app_options>` Kconfig option to enable or disable handling of the power management events (:c:struct:`power_down_event`, :c:struct:`wake_up_event`).
+This option depends on the :kconfig:option:`CONFIG_CAF_PM_EVENTS` Kconfig option and is enabled by default.
+
```

Comment:
Also, question - since this content appears 3 times across 3 articles, do we want to have a shortcut instead for easier tracking should any changes be needed?

Reply:
Discussed offline, it is parametrized (each article has different PM event related Kconfig option) so it is not that trivial to do a shortcut in this case.
Agreed that each article will have its own copy.